src/catalog/navigation-top.md

```diff
@@ -20,9 +20,38 @@ The category structure of your catalog can influence how well your site is index
 
 ## Set the top navigation
 
-See “[Creating Categories]({% link catalog/category-create.md %})” to define the category structure of the main menu. For a multistore installation, a different main menu can be assigned as the [root category]({% link catalog/category-root.md %}) for each [store]({% link stores/stores-all-create-store.md %}).
+In order for the desired category to appear in the top navigation of your store, you need to complete the following steps:
 
-## Set the depth of the top navigation
+### Step 1: Create a Category
+
+1. On the _Admin_ sidebar, go to **Catalog** > **Categories**.
+
+1. Set **Store View** to determine where the new category is to be available.
+
+1. In the category tree, select the parent category of the new category.
+
+   {:.bs-callout-info}
+   The parent is one level above the new category. Since this category name does not appear as an option in the menu, make sure your new category has Level2 and not root.
```

Comment:
See if this can be clarified and just added to the step above. Not sure if 'Level2' is correct.

Reply:
I have deleted this one, in the "Levels" clarification there is a description about the root category 
![image](https://user-images.githubusercontent.com/78729171/120534584-b0bef400-c3ea-11eb-9e8d-36fb556e1135.png)